UnicornRender/cmake/GlmConfig.cmake

```diff
@@ -5,8 +5,5 @@
 set(GLM_INSTALL_ENABLE OFF CACHE BOOL "Flag to override default GLM_INSTALL_ENABLE value")
 set(GLM_TEST_ENABLE OFF CACHE BOOL "Flag to override default GLM_TEST_ENABLE value")
 
-if (NOT DEFINED GLM_ROOT_DIR)
-    set(GLM_ROOT_DIR
-        "${UNICORN_RENDER_ROOT}/external/glm"
-        CACHE STRING "Path to GLM root directory")
-endif()
+set(GLM_ROOT_DIR "${UNICORN_RENDER_ROOT}/external/glm" CACHE STRING "Path to GLM root directory")
```

Comment:
I am not sure about this one. I will resolve this stuff in `sandbox/sleipnir` branch but I don't like that you're changing cmake logic like that.
I don't see how this is necessary and not sure how you work with `glm` anymore. I see that you've removed `add_subdirectory(${GLM_ROOT_DIR})` and I don't see where it is added anymore.
So yeah, I'm not a fan of such big changes without considering what this may affect. After `sandbox/sleipnir` integration I think all cmake changes shall be discussed explicitly before PR.

Reply:
glm is header only. Thats why Im only including this with explicit path. This was changed to remove this libglm-dev bug on ubuntu. When you will resolve and integrate sleipnir things, you can rewrite as you want under your responsibility.

Reply:
But why did you remove the `if (NOT DEFINED GLM_ROOT_DIR)` part? sure, you can get rid of `find_package` but why also change the path finding logic?